deps/pretty-print.lua

```diff
@@ -297,6 +297,16 @@ function dump(value, recurse, nocolor)
   return nocolor and strip(s) or s
 end
 
+local function console_write(fs, s)
+  s = s .. '\n'
+  repeat
+    n = uv.try_write(fs, s)
+    if n then
+      s = s:sub(n+1)
+    end
+    n = 0
```

Comment:
next line does a comparison on n equaling the length of s, but n holds the write length. There needs to be a counter.

Reply:
s = s:sub(n+1) will skip data has been write, all write will get s=''